product_pricelist_margins/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Product Pricelist Margins",
```

Comment:
module name should not end w/ plural form. Please rename it to `_margin`.
See pt 1 https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#11modules

Reply:
Hi @simahawk, I have updated it. Thanks for your suggestion